OCABundles/credentials/ongov-digital-trust/ontario-test-credential/OCABundle.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
@MosCD3 — the format is not right.  The outer brackets should be square, and what you have for line 2 is not right — the CredDef ID.